pkg/cache/cache_test.go

```diff
@@ -122,12 +123,12 @@ func TestCacheClusterQueueOperations(t *testing.T) {
 					}},
 					NamespaceSelector: labels.Nothing(),
 					FlavorFungibility: defaultFlavorFungibility,
-					Usage: FlavorResourceQuantities{
-						"default": {corev1.ResourceCPU: 0},
-					},
-					AdmittedUsage: FlavorResourceQuantities{
-						"default": {corev1.ResourceCPU: 0},
-					},
+					Usage: resources.FlavorResourceQuantitiesFlat{
+						resources.FlavorResource{Flavor: "default", Resource: corev1.ResourceCPU}: 0,
```

Comment:
I think this works?
```suggestion
						{Flavor: "default", Resource: corev1.ResourceCPU}: 0,
```

Reply:
it does, thanks!

`find ./pkg/ -type f | xargs sed -i  "s/resources.FlavorResource{/{/g"`